src/mlpack/methods/reinforcement_learning/q_learning_impl.hpp

```diff
@@ -58,14 +61,54 @@ template <
   typename NetworkType,
   typename UpdaterType,
   typename PolicyType,
-  typename ReplayType
+  typename ReplayType,
+  typename PrioritizedReplayType
+>
+QLearning<
+  EnvironmentType,
+  NetworkType,
+  UpdaterType,
+  PolicyType,
+  ReplayType,
+  PrioritizedReplayType
+>::QLearning(TrainingConfig config,
+             NetworkType network,
+             PolicyType policy,
+             PrioritizedReplayType prioritizedReplayMethod,
+             UpdaterType updater,
+             EnvironmentType environment):
+  config(std::move(config)),
+  learningNetwork(std::move(network)),
+  updater(std::move(updater)),
+  policy(std::move(policy)),
+  prioritizedReplayMethod(std::move(prioritizedReplayMethod)),
+  environment(std::move(environment)),
+  totalSteps(0),
+  deterministic(false),
+  prioritized_replay(true)
+{
+  if (learningNetwork.Parameters().is_empty())
+    learningNetwork.ResetParameters();
+  this->updater.Initialize(learningNetwork.Parameters().n_rows,
+                           learningNetwork.Parameters().n_cols);
```

Comment:
Can we tab this twice (4 spaces) to be consistent with the rest of the codebase.

Reply:
> Can we tab this twice (4 spaces) to be consistent with the rest of the codebase.

Of course, I will fix that.